examples/model_compress/amc/amc_search.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import argparse
+import time
+
+import torch
+import torch.nn as nn
+
+from nni.compression.torch import AMCPruner
+from data import get_split_dataset
+from utils import AverageMeter, accuracy
+
+device = None
```

Comment:
set device to None looks redundant, remove this line would be better?

Reply:
Thanks, removed.